experimental/traffic-portal/src/app/core/core.module.ts

```diff
@@ -74,6 +75,7 @@ export const ROUTES: Routes = [
 	{ component: TenantDetailsComponent, path: "tenants/:id"},
 	{ component: PhysLocDetailComponent, path: "phys-locs/:id" },
 	{ component: PhysLocTableComponent, path: "phys-locs" },
+	{ component: ISOGenerationFormComponent, path: "gen-iso"}
```

Comment:
The sidebar has this as `iso-gen` meaning it can't be navigated to from there

Reply:
:facepalm: